user-interface/public/nginx.conf

```diff
@@ -0,0 +1,31 @@
+server {
+    #proxy_cache cache;
+    #proxy_cache_valid 200 1s;
+    listen 8080;
+    listen [::]:8080;
+
+    root /home/site/wwwroot;
+
+    index index.html index.htm;
+    server_name  example.com www.example.com;
```

Comment:
This probably doesn't matter but curious to know why we left the server name "example".

Reply:
At the moment, I did not what to put there.

Reply:
Because there is only one server block define, the value for `server_name` should not matter.